man/ukify.xml

```diff
@@ -498,6 +499,36 @@ Phases=enter-initrd:leave-initrd
       <para>This creates a signed PE binary that contains the additional kernel command line parameter
       <literal>debug</literal> with SBAT metadata referring to the owner of the addon.</para>
     </example>
+
+    <example>
```

Comment:
hmmm, so you add the example here, but never document the "genkey" verb in the first place...